client/angular/controllers/AdminController.js

```diff
@@ -1,3 +1,4 @@
+/*global console*/
```

Comment:
You can take out all the "global console" declarations and just add console as a global in the .jshintrc, otherwise looks good